app/controllers/publishers/vacancies_controller.rb

```diff
@@ -59,8 +61,25 @@ def summary
     @feedback_form = Publishers::JobListing::FeedbackForm.new
   end
 
+  def applications
+    @vacancy = VacancyPresenter.new(@vacancy)
+    @application_statuses = @vacancy.job_applications.group_by { |k| k[:status] }.transform_values(&:count)
```

Comment:
It would save 1 reloading of the applications from the database if you call your `vacancy_applications` method here instead of `@vacancy.job_applications`.